### ISSUE TITLE
No HTTP & HTTPS Request on Windows7 Guest

### ISSUE THREAD

**Issue by MeteorAdminz** [cuckoosandbox/cuckoo#828]
Hello, 
I have 2 VM's, 1 is windows XP and another 1 is windows7. On the windows XP, I can get full pcap with HTTP request. But on the windows 7, I only can get host and UDP and no HTTP request. The network that I used are same with windows XP which is vboxnet0 only the IP is different. windows XP: 192.168.56.101, windows7: 192.168.56.102, host: ubuntu14.04 (LTS). Has anyone got the same problem with me?

**Comment by jbremer** [comment]
Could you share both pcap files, please?

**Comment by MeteorAdminz** [comment]
[pcap_dump](https://www.dropbox.com/s/aagfwfencv1ix8d/pcap_dump.zip?dl=0), here you go. sorry for the late reply. :(

**Comment by jbremer** [comment]
It seems like the traffic doesn't find its way back to your second machine - it doesn't even resolve DNS queries properly, so there's something incorrect with your setup, probably ;-)

**Comment by MeteorAdminz** [comment]
Thanks for your response. :)
Pretty weird, because this only happened to windows 7. On Windows XP, I have tried to create multiple VM's but I can get the HTTP & HTTPS response and I also try the IP that used for windows 7 and the result also same. I also tried to remove all the VM's and only windows 7 are left. The setting that I used for windows XP are all applied to windows 7 but I still can't get the HTTP & HTTPS request.
